templates/zerver/emails/missed_message.source.html

```diff
@@ -25,14 +25,14 @@
 {% block manage_preferences %}
 <div class="email-preferences">
     &mdash;<br />
-    {% if mention %}
-    {% if mentioned_user_group_name %}
-    {% trans %}You are receiving this because @{{ mentioned_user_group_name }} was mentioned in {{ realm_name }}.{% endtrans %}<br />
-    {% else %}
-    {% trans %}You are receiving this because you were mentioned in {{ realm_name }}.{% endtrans %}<br />
-    {% endif %}
+    {% if personal_mentioned %}
+    {% trans %}You are receiving this because you were personally mentioned.{% endtrans %}<br />
+    {% elif mentioned_user_group_name %}
+    {% trans %}You are receiving this because @{{ mentioned_user_group_name }} was mentioned.{% endtrans %}<br />
+    {% elif wildcard_mentioned %}
+    {% trans %}You are receiving this because everyone was mentioned.{% endtrans %}<br />
```

Comment:
I think it'd be nice to have this be `because everyone was mentioned in #{{stream_name}}.` -- since that's likely fairly important information for interpreting the notice.

Reply:
I updated it.

Reply:
Sounds reasonable. Could you please update the screenshot in the PR description?

Reply:
I updated screenshot.